lib/msal-node/test/client/ManagedIdentitySources/MachineLearning.spec.ts

```diff
@@ -80,16 +86,24 @@ describe("Acquires a token successfully via an Machine Learning Managed Identity
             const url: URLSearchParams = new URLSearchParams(
                 sendGetRequestAsyncSpy.mock.lastCall[0]
             );
+            expect(
+                url.has(
+                    ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID
+                )
+            ).toBe(false);
```

Comment:
Can you explain what `MANAGED_IDENTITY_CLIENT_ID` and `MANAGED_IDENTITY_CLIENT_ID_2017` denotes?